src/components/Folder.css

```diff
@@ -0,0 +1,91 @@
+.Folder {
+  position: absolute;
+  top: 40px;
+  left: 70px;
+  bottom: 0;
+  width: 300px;
+  padding-top: 20px;
+  background: #2e2e2e;
+  border-right: 1px solid black;
+  text-align: center;
+  transform: translateX(-100%);
+  transition: 0.2s transform;
+  color: #fff;
+}
+
+.Folder-expanded {
+  transform: translateX(0);
+  transition: 0.2s transform;
+}
+
+.Folder-saved-list {
+  margin-top: 10px;
+  padding: 10px 10px;
+}
+
+.Folder-saved-item {
+  display: flex;
+  margin-top: 5px;
+  text-align: start;
+}
+
+.Folder-saved-item:hover {
+  cursor: pointer;
+}
+
+.Folder-previous-items {
+  padding-top: 30px;
+}
+
+.Folder-item-text {
+  display: block;
+  text-align: start;
+  padding-left: 10px;
+}
+
+.Folder-small-text {
+  font-size: 0.75rem;
+}
+
+.Folder-input {
+  height: 20px;
+  width: 150px;
+  margin-bottom: 15px;
+  margin-top: 5px;
+  border: 1px solid #000;
+  font-size: 1em;
+  border-radius: 5px;
+  padding: 5px;
+  outline: none !important;
+  text-align: start;
+}
+
+.Folder-input-error {
```

Comment:
This class exists (good), but I don't see any validation/error state on the input itself?

Reply:
now used in input validation